x-pack/libbeat/Jenkinsfile.yml

```diff
@@ -33,3 +34,7 @@ stages:
             tags: true         ## for all the tags
     build:
         mage: "mage build test"
+        when:                  ## Override the top-level when.
+            not_changeset_full_match: "^libbeat/.*" ## Disable the stage if ONLY changes for the OSS beats
```

Comment:
If there are changes in the code of libbeat we should run the tests for x-pack/libbeat. I guess this is done now by `@ci` or `GetProjectDependencies`, but will this new condition prevent tests to be run if there are changes only in OSS?

Reply:
Aha! Then this is totally a different behaviour that I though! good catch. The `@xpack` macro already contain the `"^libbeat/.*"`, therefore we don't need this when